plugins/banners/list.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+// const boom = require('boom');
+const schema = require('screwdriver-data-schema');
+const listSchema = schema.models.banner.list;
+
+module.exports = () => ({
+    method: 'GET',
+    path: '/banners',
+    config: {
+        description: 'Get banners',
+        notes: 'Returns all banner records',
+        tags: ['api', 'banner'],
+        auth: {
+            strategies: ['token'],
+            scope: ['user']
+        },
+        plugins: {
+            'hapi-swagger': {
+                security: [{ token: [] }]
+            }
+        },
+        handler: (request, reply) => {
+            const { bannerFactory } = request.server.app;
+
+            const config = Object.assign({}, request.payload);
```

Comment:
don't need this line

Reply:
as discussed in the meeting today, we won't be implementing pagination in banners, at least not yet

Reply:
Regardless, this line won't work. There shouldn't be a `payload` in a GET request.

It can be changed to 
```
const config = request.params ? { params: request.params } : {};
```